Gemfile

```diff
@@ -5,6 +5,7 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
+gem 'twilio-ruby'
```

Comment:
When you add a gem, it's always good to specify a version, like this:

`gem 'twilio-ruby', '~> 5.0.0.rc21'`

The `~>` tells bundler that it's allowed to automatically update the patch level, but not the minor or major. That keeps you safe from api changes in libraries that follow [semantic versioning](http://semver.org/)